docs/src/specification/encoding.rst

```diff
@@ -41,6 +53,12 @@ These structures are defined as follows:
 
     RegisterPrefix = REGISTER-PREFIX-TYPE TLV-LENGTH Name
 
+    SyncPrefix = SYNC-PREFIX-TYPE TLV-LENGTH Name
+
+    DataNameDedupe = SYNC-DATA-NAME-DEDUPE-TYPE TLV-LENGTH ; TLV-LENGTH = 0
```

Comment:
Just a reminder this is pure hack for NDN Workspace. Ideally we should let apps define naming conventions.